clowdapp.yaml

```diff
@@ -57,6 +57,10 @@ objects:
             value: ${CLOWDER_ENABLED}
           - name: ENABLE_RBAC
             value: "${ENABLE_RBAC}"
+          - name: GARBAGE_COLLECTION_INTERVAL
```

Comment:
Should we consider adding these to processor container only?

Reply:
Added on processor side only to avoid confusion! Thanks.